### PR TITLE
Release 2019-10-15: versions & changelogs update

### PR DESCRIPTION
**Pull request by msJinLei**
## Checklist

- [ ] I have read the [_Submitting Changes_](../CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

File: ChangeLog.md [hunk 1]
```diff
@@ -1,3 +1,71 @@
+## 2.8.0 - October 2019
```
Comment (erich-wang) [inline ChangeLog.md]: Normally need to add general section when having new module release
```
### General
* Az.HealthcareApisService 1.0.0 release
```

Comment (msJinLei) [reply ChangeLog.md]: what about the change list of that module?
The change log should be 
### General
* Az.HealthcareApisService 1.0.0 release
  * Updated the powershell version to 1.0.0
  * Updated the SDK version to 1.0.2
  * Update in tests to refer to new SDK version
  * Updated the output structure from nested to flattened.
or just
### General
* Az.HealthcareApisService 1.0.0 release

**Comment by isra-fel** [comment]
Hi @msJinLei sql module needs to be regenerated, see:
https://github.com/Azure/azure-powershell/blob/release-2019-10-15/src/Sql/Sql/ChangeLog.md

File: src/HealthcareApis/HealthcareApis/ChangeLog.md [hunk 2]
```diff
@@ -18,6 +18,8 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+
+## Version 0.1.3
```
Comment (markcowl) [inline src/HealthcareApis/HealthcareApis/ChangeLog.md]: Need to update this to 1.0.0

Comment (markcowl) [reply src/HealthcareApis/HealthcareApis/ChangeLog.md]: There should also be updates to AssemblyInfo.cs

File: tools/Tools.Common/SerializedCmdlets/Microsoft.Azure.PowerShell.Cmdlets.HealthcareApis.dll.json [hunk 3]
```diff
@@ -0,0 +1,4239 @@
...
+      "SupportsShouldProcess": false,
+      "ConfirmImpact": 2,
+      "SupportsPaging": false,
+      "DefaultParameterSetName": "ListParameterSet",
+      "OutputTypes": [
+        {
+          "Type": {
+            "Namespace": "Microsoft.Azure.Commands.HealthcareApis.Models",
+            "Name": "Microsoft.Azure.Commands.HealthcareApis.Models.PSHealthcareApisService",
+            "AssemblyQualifiedName": "Microsoft.Azure.Commands.HealthcareApis.Models.PSHealthcareApisService, Microsoft.Azure.PowerShell.Cmdlets.HealthcareApis, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null",
```
Comment (markcowl) [inline tools/Tools.Common/SerializedCmdlets/Microsoft.Azure.PowerShell.Cmdlets.HealthcareApis.dll.json]: This looks correct

**Comment by isra-fel** [comment]
> Hi @msJinLei sql module needs to be regenerated, see:
> https://github.com/Azure/azure-powershell/blob/release-2019-10-15/src/Sql/Sql/ChangeLog.md

Guess it'll be updated in the next run. Nevermind

File: ChangeLog.md (hunk 1 above)
Comment (erich-wang) [reply ChangeLog.md]: yes

File: ChangeLog.md [hunk 4]
```diff
@@ -1,3 +1,80 @@
...
+### General
+* Az.HealthcareApis 1.0.0 release
+
+#### Az.Accounts
+* Fixed miscellaneous typos across module
+* Support user-assigned MSI in Azure Functiosn Authentication (#9479)
```
Comment (erich-wang) [inline ChangeLog.md]: Is there any change for Az.Accounts?

File: src/HealthcareApis/HealthcareApis/Az.HealthcareApis.psd1 [hunk 5]
```diff
@@ -3,7 +3,7 @@
 #
```
Comment (erich-wang) [inline src/HealthcareApis/HealthcareApis/Az.HealthcareApis.psd1]: Need update AssemblyInfo.cs for HealthcareApis

File: src/SqlVirtualMachine/SqlVirtualMachine/Az.SqlVirtualMachine.psd1 [hunk 6]
```diff
@@ -3,7 +3,7 @@
 #
```
Comment (erich-wang) [inline src/SqlVirtualMachine/SqlVirtualMachine/Az.SqlVirtualMachine.psd1]: Miss AssemblyInfo.cs for in SqlVirtualMachine?

**Pull request closed without merging**